server/src/tenant/schema-builder/schema-builder.service.ts

```diff
@@ -156,6 +156,11 @@ export class SchemaBuilderService {
     const mutationFields: any = {};
 
     for (const objectDefinition of objectMetadata) {
+      if (objectDefinition.fields.length === 0) {
```

Comment:
I had an error when creating only the object without any fields. I think the schema should still be able to generate the type with the default fields such as id, createdAt, updatedAt, deletedAt. I'll need to check this implementation, it might use only what's stored inside field-metadata and fails since we don't store default fields as metadata